pytest_fixtures/core/broker.py

```diff
@@ -202,6 +202,13 @@ def module_capsule_configured_mqtt(module_capsule_configured):
     result = module_capsule_configured.execute('firewall-cmd --permanent --add-port="1883/tcp"')
     assert result.status == 0, 'Failed to open mqtt port on capsule'
     module_capsule_configured.execute('firewall-cmd --reload')
+    # lower the mqtt_resend_interval interval
+    # TODO use installer command instead once merged downstream
+    module_capsule_configured.execute(
+        "echo ':mqtt_resend_interval: 30' >> /etc/foreman-proxy/settings.d/remote_execution_ssh.yml"
+    )
+    result = module_capsule_configured.execute('systemctl restart foreman-proxy')
+    assert result.status == 0, 'foreman-proxy restart unsuccessful'
```

Comment:
Would suggest using f-m CLI here for restart instead of systemd
```suggestion
    result = module_capsule_configured.cli.Service.restart(options={'only': 'foreman-proxy'})
    assert result.status == 0, 'foreman-proxy restart unsuccessful'
```